### DISCUSSION TITLE
GridCellKind.Uri display innerText seperate from href

### DISCUSSION THREAD

**Discussion by coffeepy** [glideapps/glide-data-grid#566] [Q&A] [answered]
Is it possible without needing to render a custom cell to use the GridCellKind.Uri with a displayData key to have a certain text string displayed instead of the URL, just like your a tag in html? or perhaps a different way?

Want the same behavior as this "a" element below
`<a href="https://google.com">Here is a link to Google</a>`

**Comment by jassmith** [comment, marked as answer]
great feature suggestion, adding now.

**Reply by jassmith** [reply]
Done

**Reply by coffeepy** [reply]
Thanks so much! That was quick :)
